x-pack/plugins/security_solution_serverless/public/upselling/sections/rule_details_endpoint_exceptions/index.tsx

```diff
@@ -0,0 +1,14 @@
+/*
```

Comment:
same here. If our team is owner, please move it to the existing `endpoint_management` folder

Reply:
a9751c61a3cbde0a7c9a8da5e764b8a482003b36